### ISSUE TITLE
It won't finish exporting photos/videos

### ISSUE THREAD

**Issue by SolidCapo** [damonlynch/rapid-photo-downloader#70]
It stays in 95-99%, saying that it only have a few seconds to finish (and the speed in MB/s decreases to 0 veeeery slowly) and have to close the app and reopen each time i export.

![imagen](https://user-images.githubusercontent.com/57196362/156224390-f85e4913-addb-44b3-a435-325e3d17abab.png)
It finally reached 0.0 MB/s and still won't finish :( Look:
![imagen](https://user-images.githubusercontent.com/57196362/156224623-44fe9344-c141-462d-98d0-d6f09c0a41ee.png)

Here's the file:
[rpd-bug-report-20220301.tar.gz](https://github.com/damonlynch/rapid-photo-downloader/files/8163824/rpd-bug-report-20220301.tar.gz)

**Comment by damonlynch** [comment]
Is this with similar files that you already uploaded to Google Drive?

**Comment by damonlynch** [comment]
I installed Manjaro. I used your config file and the Sony files you provided in Google Drive. I could not replicate the problem. However, I recommend either waiting until 0.9.30 is in Manjaro because otherwise you will immediately see the crash you found in #69 . However, if you want to use the 0.9.29 in the meantime, set the environment variable RPD_DEVEL_DEFAULTS and launch the program from the command line like this:

```
RPD_DEVEL_DEFAULTS=1 rapid-photo-downloader
```

**Comment by SolidCapo** [comment]
> Is this with similar files that you already uploaded to Google Drive?

Yes, those files are only a few of the session, but it happens with any files :face_exhaling: 

> ause otherwise you will immediately see the crash you found in #69 . However, if you want to use the 0.9.29 in the meantime, set the environ

I was going to do this, but just opened it and says that there's a new version, so, i'll update it and see what happens :)

**Comment by damonlynch** [comment]
Thank you
